CMakeLists.txt

```diff
@@ -230,6 +230,9 @@ set(exec_prefix "\${prefix}")
 set(includedir "\${prefix}/include")
 set(libdir "\${exec_prefix}/lib")
 configure_file(quantlib-config.in quantlib-config @ONLY)
+configure_file(quantlib.pc.in quantlib.pc @ONLY)
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/quantlib-config TYPE BIN)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/quantlib.pc DESTINATION lib/pkgconfig)
```

Comment:
is DESTINATION relative to the prefix?

Reply:
Seems to be, according to the [docs](https://cmake.org/cmake/help/latest/command/install.html) and a few rounds of local testing:

> DESTINATION `<dir>`
> Specify the directory on disk to which a file will be installed. Arguments can be relative or absolute paths.
> 
> If a relative path is given it is interpreted relative to the value of the [CMAKE_INSTALL_PREFIX](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html#variable:CMAKE_INSTALL_PREFIX) variable. The prefix can be relocated at install time using the DESTDIR mechanism explained in the [CMAKE_INSTALL_PREFIX](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html#variable:CMAKE_INSTALL_PREFIX) variable documentation.

Reply:
Ok, thanks